emcc.py

```diff
@@ -3427,8 +3434,7 @@ def module_export_name_substitution():
     src = src.replace(shared.JS.module_export_name_substitution_pattern, replacement)
     # For Node.js and other shell environments, create an unminified Module object so that
     # loading external .asm.js file that assigns to Module['asm'] works even when Closure is used.
-    if shared.Settings.MINIMAL_RUNTIME and (shared.Settings.target_environment_may_be('node') or
-                                            shared.Settings.target_environment_may_be('shell')):
+    if shared.Settings.MINIMAL_RUNTIME and not shared.Settings.MODULARIZE_INSTANCE and (shared.Settings.target_environment_may_be('node') or shared.Settings.target_environment_may_be('shell')):
```

Comment:
why does modularize_instance matter here? (and, why not modularize?)

Reply:
MODULARIZE_INSTANCE creates the Module on the next line, emitting

```
if(typeof Module==="undefined"){var Module={};}
var Module = (
```
would be redundant for code size.